rest/src/plugins/mosaicRestrictions/mosaicRestrictionsRoutes.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2016-present,
+ * Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp. All rights reserved.
+ *
+ * This file is part of Catapult.
+ *
+ * Catapult is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Catapult is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Catapult.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const routeUtils = require('../../routes/routeUtils');
+const errors = require('../../server/errors');
+const AccountType = require('../AccountType');
+const catapult = require('catapult-sdk');
+
+const { address, networkInfo, mosaicRestriction } = catapult.model;
+const { uint64 } = catapult.utils;
+
+module.exports = {
+	register: (server, db, services) => {
+		const accountIdToAddress = (type, accountId) => ((AccountType.publicKey === type)
```

Comment:
same as metadata, im unsure if this is unique here or if it can be reused, either way, move to `routeUtils` (after merge)